src/types/lib.ts

```diff
@@ -2,3 +2,11 @@ export interface ClientOptions {
 	token?: string;
 	intents: number;
 }
+
+export const enum RequestMethod {
+	Delete = 'DELETE',
+	Get = 'GET',
+	Patch = 'PATH',
```

Comment:
Ah yes, good ol PATH

```suggestion
	Patch = 'PATCH',
```